index.html

```diff
@@ -2,9 +2,58 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="vendor/icon.css">
   </head>
   <body>
+    <nav>
```

Comment:
Also, don't forget to add your name to your header so the person on your site knows who you are :)